SPECS/libmspack/libmspack.spec

```diff
@@ -1,20 +1,21 @@
 Summary:        A library that provides compression and decompression of file formats used by Microsoft
 Name:           libmspack
-Version:        0.7.1alpha
-Release:        4%{?dist}
+Version:        0.10.1alpha
```

Comment:
Is there a non-alpha version available?

Reply:
No, all of its versions are tagged with alpha at the end